File: app/src/main/java/gateway/hitrontech/com/encryption/fragment/about/AboutFragment.java
package gateway.hitrontech.com.encryption.fragment.about;

import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import gateway.hitrontech.com.encryption.R;
import gateway.hitrontech.com.encryption.base.BaseFragment;
import gateway.hitrontech.com.encryption.databinding.FragmentAboutBinding;
import gateway.hitrontech.com.encryption.utils.FileUtils;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public class AboutFragment extends BaseFragment {

  private FragmentAboutBinding mBinding;

  public static Fragment getInstance() {
    AboutFragment fragment = new AboutFragment();
    Bundle args = new Bundle();
    fragment.setArguments(args);
    return fragment;
  }

  @NotNull
  @Override
  protected View createView(@NotNull LayoutInflater inflater, @Nullable ViewGroup container,
      @Nullable Bundle savedInstanceState) {

    if (!getViewExisted()) {
      mBinding = DataBindingUtil.inflate(inflater, R.layout.fragment_about, container, false);
    }
    return mBinding.getRoot();
  }

  @Override
  protected void viewFound(@Nullable View view) {
    if (!getViewExisted()) {
      mBinding.filePath.setText(FileUtils.getResultPath());
    }
  }

  @NotNull
  @Override
  protected String getTitle() {
    return "关于";
  }
}

File: app/src/main/java/gateway/hitrontech/com/encryption/fragment/about/AboutFragment.kt
package gateway.hitrontech.com.encryption.fragment.about

import android.databinding.DataBindingUtil
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import gateway.hitrontech.com.encryption.R
import gateway.hitrontech.com.encryption.base.BaseFragment
import  gateway.hitrontech.com.encryption.databinding.FragmentAboutBinding
import gateway.hitrontech.com.encryption.utils.FileUtils

class AboutFragment : BaseFragment() {

    private var mBinding: FragmentAboutBinding? = null

    override fun createView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        if (!viewExisted) {
            mBinding = DataBindingUtil.inflate(inflater, R.layout.fragment_about, container, false)
        }
        return mBinding!!.root
    }

    override fun viewFound(view: View?) {
        if (!viewExisted) {
            mBinding!!.filePath.text = FileUtils.resultPath
        }
    }

    override fun getTitle() = "关于"

    companion object {
        fun getInstance(): AboutFragment {
            val fragment = AboutFragment()
            val args = Bundle()
            fragment.arguments = args
            return fragment
        }
    }

}